src/components/LocationPage/ChartsHolder.tsx

```diff
@@ -147,6 +149,9 @@ const ChartsHolder = ({ projections, region, chartId }: ChartsHolderProps) => {
           isMobile={isMobile}
           region={region}
         />
+        <MainContentInner>
+          <VaccineDetails region={region} />
+        </MainContentInner>
```

Comment:
Avoid rendering `MainContentInner` when there isn't vaccination data, otherwise it might show an empty frame

```tsx
{vaccineInfo && (
    <MainContentInner>
      <VaccineDetails region={region} vaccineInfo={vaccineInfo} />
    </MainContentInner>
)}
```